README.md

```diff
@@ -22,12 +22,16 @@ const vast = new VAST()
 
 Automatically generated API documentation can be found on
 [GitHub Pages](https://zentrick.github.io/iab-vast-model/). You can also
-open `docs/index.html` in a Web browser. Running `gulp doc` will (re)build the
+open `docs/index.html` in a Web browser. Running `npm run doc` will (re)build the
 `docs` directory.
 
 Additionally, feel free to look at the source code. We recommend starting with
 `src/index.js`, which should export all the classes you need.
 
+## Typing
+
+All APIs are statically typed using (flow)[https://flow.org]. Type checking is automatically performed when running `npm test`. You can also run type checking only by using the command `npm run flow`. Static typing adds an additional layer of defense against errors and can also improve the development experience by enabling autocompletion. For (atom.io)[https://atom.io], we recommend (Flow-IDE)[https://atom.io/packages/flow-ide].
```

Comment:
I'd leave out the last two sentences and keep the readme lean and mean. It's not our job to educate people on static typing or to impose an editor.